src/pages/create.tsx

```diff
@@ -129,48 +132,40 @@ const CreateProgramPage: Page = () => {
 
   const stepTwo = () => {
     return (
-      <div className="flex flex-col w-full px-4 py-12 md:w-120">
-        <Text h1 b>
-          Program Details
-        </Text>
-        <Text>
-          Let's set up some basic information! You can also do these later.
-        </Text>
-        <div className="h-6" />
+      <div className="flex flex-col space-y-8">
+        <div>
+          <Text h1 b>
+            Program Details
+          </Text>
+          <div className="h-2" />
+          <Text className="text-secondary">
+            Let's set up some basic information! You can also do these later.
+          </Text>
+        </div>
         <div>
           <Text b>Logo</Text>
-          <div className="h-1" />
-          <Button variant="inverted" size="small">
-            Choose File
-          </Button>
-          <input // TODO: display preview of logo
-            type="file"
-            name="Program Logo"
-            value={programLogo}
-            className="flex-1"
-            onChange={(e) => {
-              setProgramLogo(e.target.value);
-            }}
-          />
+          <div className="h-2" />
+          <InputPreview setFile={setProgramLogo} />
         </div>
 
-        <div className="h-6" />
-        <Text b>Identifier</Text>
-        <div className="inline">
-          <Text b2>www.mentorcenter.us/</Text>
-          <Input
-            title="Identifier"
-            name="Program Identifier"
-            placeholder="nova-mentorship"
-            value={programIdentifier}
-            className="flex-1"
-            onChange={(e) => {
-              setProgramIdentifier(e.target.value);
-            }}
-          />
+        <div>
+          <Text b>Identifier</Text>
+          <div className="h-2" />
+          <div className="inline">
+            <Text b2>www.mentorcenter.us/</Text>
+            <Input
+              title="Identifier"
+              name="Program Identifier"
+              placeholder="nova-mentorship"
+              value={programIdentifier}
+              className="flex-1"
```

Comment:
Stylign comment but I feel like padding is too large. Maybe p-1? Or whatever looks better